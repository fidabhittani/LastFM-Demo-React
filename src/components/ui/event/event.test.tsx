import React from "react";
import { shallow } from "enzyme";
import Event from "./index";

/** Event TEST */
describe("Event Root Component Tests", () => {
  const event = {
    id: 2,
    event_name: "Voyatouch",
    country: "Nigeria",
    city: "Shani",
    venue: "46209 Nelson Hill",
    date: "18-Aug-2019"
  };

  it("Event Component renders correctly", () => {
    const eventWrapper = shallow(<Event event={event} />);
    expect(eventWrapper).toMatchSnapshot();
  });
});
