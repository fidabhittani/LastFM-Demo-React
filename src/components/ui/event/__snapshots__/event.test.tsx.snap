// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Event Root Component Tests Event Component renders correctly 1`] = `ShallowWrapper {}`;
